### DISCUSSION TITLE
Is there any licensing issue when we share ImageMagick with our team?

### DISCUSSION THREAD

**Discussion by hide415** [ImageMagick/ImageMagick#3240] [Help]
I would like to put the portable version of ImageMagick under a version control system such as svn or perforce, and share it within the company.
We would also like to share this environment with our partner companies.
Is there any problem in licensing?

**Comment by urban-warrior** [comment]
Have you read https://imagemagick.org/script/license.php?  Its a pretty liberal license and provides guidance for its use.

**Reply by hide415** [reply]
Yes, I read the license description above.
I understand that it is a fairly liberal license, but I was wondering if you could give me some guidance on how to provide it in the way I have suggested.
I look forward to hearing from you.

**Comment by urban-warrior** [comment]
We have nothing to add over what is captured in https://imagemagick.org/script/license.php.

**Reply by hide415** [reply]
Thank you for your reply.
I will read the site and try to understand more about licensing.
Thank you very much for taking the time to do this.
